### ISSUE TITLE
ArchiPack Pro 2.5.1 crashes in Blender 3.3.0 on MacBook Pro M1

### ISSUE THREAD

**Issue by HerrSchulzGermany** [s-leger/archipack#340]
Archipack version:
2.5.1
Blender version:
3.3.0
Os version:
Mac OS Monterey 12.6
Description of the issue:
ArchiPack Pro 2.5.1 crashes in Blender 3.0 on MacBook Pro M1 In conjunction with the following addons enabled:
BlenderKit
Boxcutter 
HardOps 9
Easy HDRI 
KitOps
KitOps BEVEL
KitOps Toggle VP Display
PowerBackup
PowerSave
SimpleTabs
ArchiPack Pro - when disabled, there is no crash anymore
Bolt Factory 
Geodesign Domes 
Qshot2
CinePack2
TriLightning
Extreme PBR 
ABS Plastik Materials
LoopTools
DECALMachine
Machine3Tools
MeshMachine
CableRator
ChalkStyle Pro
MiraTools

If needed I can reproduce a crashreport - feel free to send me an email.

Thomas

**Comment by s-leger** [comment]
Hi Thomas,
In order to narrow down the issue, you should enable addon filtering in the workspace settings, and then use a "binary method" by enabling only half of the addons.
Basically enable half of the addons, and see if the conflict lie with that half or the other one, then repeat until you find the one.

**Comment by HerrSchulzGermany** [comment]
Hi,

For you to know - it’s crashing when I quit the blender. Not while working.
Anyway I will do as follows. I’ll disable all 3rd parties except of ArchipackPro and enable them one after the other.
I will writ every Step in a *.txt and send it as an attachment, or if it’s not so much directly in the thread


Cheers, Thomas

**Comment by HerrSchulzGermany** [comment]
I made my homework ;-)

All 3rd Party Addons disabled

**With ArchiPack enabled I enable one after the other**
BoxCutter - no crash
EasyHDRI - no
HardOps9 - no
KitOps - no
KitOps BEVEL - no
KitOps Toggle VP Display - no
PowerBackup - no
PowerSave - no
SimpleTabs - no
BoltFactory - no
GeoDesic Domes - no
QShot - no
CinePack V2 - no
TriLightning - no

_ExtremePBR - crash for testing disabled ArchiPack - no crash
Now I enable Archipack and disable ExtremePBR and follow along with enabling the others_

ABS Plastic Materials - no

_DecalMachine - crash / Same procedure as ExtremePBR - same result_

MeshMACHIN3 Tools - no

_MeshMachine - crash / Same procedure as ExtremePBR - same result
Cablerator - crash / Same procedure as ExtremePBR- same result_

OCD - no
ChalkStyle Pro - no
Mira Tools - no

**In summary I can say for my configuration Mac and blender as far as I can test it
blender crashes on quit (not while working) with ArchiPack PRO enabled when I enable one of
the above listed Addons**

Extreme PBR
DecalMachine
MeshMachine
Cablerator

Now I will disable ArchiPack PRO and enable the 4 above. One after the other.

**ArchiPack Pro disabled**

enabling
Extreme PBR - no
DecalMachine - no
MeshMachine - no
Cablerator - no


Feel free to tell me what you want me to test

Thomas
